nimble/include/nimble/ble.h

```diff
@@ -109,9 +109,9 @@ struct ble_mbuf_hdr_rxinfo
 struct ble_mbuf_hdr_txinfo
 {
     uint8_t flags;
-    uint8_t offset;
-    uint8_t pyld_len;
     uint8_t hdr_byte;
+    uint16_t offset;
+    uint16_t pyld_len;
```

Comment:
Changing the types to uint16_t here breaks larger replies from a peripheral. For instance listing the images using newtmgr with two images present on the peripheral. Changing the types back to uint8_t makes it work. Adding a dummy byte before the offset / pyld_len breaks download again. I.e.:

```
/* This works */
struct ble_mbuf_hdr_txinfo
 {
     uint8_t flags;
     uint8_t offset;
     uint8_t pyld_len;
     uint8_t hdr_byte;
}; 
```
```
/* This doesn't work */
struct ble_mbuf_hdr_txinfo
{
    uint8_t flags;
    uint8_t offset;
    uint8_t dummy;
    uint8_t pyld_len;
    uint8_t hdr_byte;
}; 
```

Is anyone else seeing this? @andrzej-kaczmarek